addons/patches/a3_static/cfgVehicles.inc

```diff
@@ -0,0 +1,21 @@
+class LandVehicle;
+class StaticMGWeapon;
+class HMG_01_base_F;
+class GMG_01_base_F;
```

Comment:
Put L1-L4 inside of `class CfgVehicles { }` and this is all set. Sorry for the delay, been busy with work.

Nit—generally I capitalize the file name `CfgVehicles` but windows has case-sensitivity issues so don't worry about trying to change it.